src/cmd/preview.go

```diff
@@ -15,17 +15,23 @@ import (
 // previewCmd represents the preview command
 var previewCmd = &cobra.Command{
 	Use:   "preview [SAMPLES_COUNT]",
-	Short: "Preview the data found in your Kubernetes cluster returning SAMPLES_COUNT",
+	Short: "Preview the data found in your Kubernetes cluster",
 	Long: `This command will print out all the data it can find in your Kubernetes cluster based on the settings in the configuration file.
+By default a randomly selected sample will be shown.
 If the optional argument SAMPLES_COUNT=0 this will print out everything.`,
 	Args:       cobra.MaximumNArgs(1),
-	ArgAliases: []string{"samples"},
+	ArgAliases: []string{"SAMPLES_COUNT"},
 	Run: func(cmd *cobra.Command, args []string) {
-		sampleCount := 5
-		if len(args) > 0 {
-			if parsedSamples, err := strconv.Atoi(args[0]); err == nil {
-				sampleCount = parsedSamples
+		var (
+			err         error
+			sampleCount int = 5
+		)
+		if len(args) == 1 {
+			sampleCount, err = strconv.Atoi(args[0])
+			if sampleCount < 0 {
+				err = fmt.Errorf("SAMPLES_COUNT must be >= 0, got %d", sampleCount)
 			}
```

Comment:
This is unnecessary logic the func PrintServices down below handles this

Reply:
Are you referring specifically to the check that looks at `if sampleCount < 0`? 

I would prefer that we reject any values set by the user that don't make sense even at read time if they are handled by functions they are being passed to. 

But if you still think it's worth removing we can do so since as you pointed out there's no harm in removing it.

Reply:
@rocktavious the validation code i added to `preview` and `reconcile` has been removed. Like you said we are using defensive programming in all functions that consume these inputs.